LogosLinuxInstaller.py

```diff
@@ -80,6 +83,14 @@ def parse_args(args):
     if args.reinstall_dependencies:
         config.REINSTALL_DEPENDENCIES = True
 
+    if args.get_winetricks: 
+       setWinetricks
+       sys.exit(0)
```

Comment:
function needs to be called with `setWinetricks()`

Reply:
Fixed in current draft, but per our discussion, this should probably set a var that we later evaluate.